tcf_website/models/models.py

```diff
@@ -382,6 +430,78 @@ class Meta:
         ]
 
 
+class CourseGrade(models.Model):
+    course = models.ForeignKey(Course, on_delete=models.CASCADE, null=True)
+    subdepartment = models.CharField(max_length=255)
+    number = models.IntegerField(
+        validators=[
+            MinValueValidator(0),
+            MaxValueValidator(99999)],
+        default=0)
+    title = models.CharField(max_length=225, default="")
+    average = models.FloatField(default=0.0)
+    a_plus = models.IntegerField(default=0)
+    a = models.IntegerField(default=0)
+    a_minus = models.IntegerField(default=0)
+    b_plus = models.IntegerField(default=0)
+    b = models.IntegerField(default=0)
+    b_minus = models.IntegerField(default=0)
+    c_plus = models.IntegerField(default=0)
+    c = models.IntegerField(default=0)
+    c_minus = models.IntegerField(default=0)
+    d_plus = models.IntegerField(default=0)
+    d = models.IntegerField(default=0)
+    d_minus = models.IntegerField(default=0)
+    f = models.IntegerField(default=0)
+    ot = models.IntegerField(default=0)
+    drop = models.IntegerField(default=0)
+    withdraw = models.IntegerField(default=0)
+    total_enrolled = models.IntegerField(default=0)
+
+    def __str__(self):
+        return f"{self.subdepartment} {self.number} {self.average}"
+
+
+class CourseInstructorGrade(models.Model):
+    instructor = models.ForeignKey(
+        Instructor, on_delete=models.CASCADE, null=True)
+    first_name = models.CharField(max_length=225)
```

Comment:
Are these needed if already instructor ForeignKey?

Reply:
yeah that's what i thought; it's part of the CSV data but i guess they made the script to do it